docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/client/DockerCmdExecConfig.java

```diff
@@ -0,0 +1,38 @@
+package com.nirima.jenkins.plugins.docker.client;
+
+import com.nirima.jenkins.plugins.docker.DockerCloud;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+import java.io.Serializable;
+
+/**
+ * Serializable object that store options for {@see com.github.dockerjava.jaxrs.DockerCmdExecFactoryImpl}
+ * Required for {@see #DockerBuilderPublisher.class} that builds DockerClient on slave side
+ *
```

Comment:
@lanwen i added description, though i think it not the end of story with this client/config builders...